code/obj/item/clothing/glasses.dm

```diff
@@ -423,22 +423,21 @@
 
 	attack(obj/item/W, mob/M)
 		if(istype(W, /mob/living/critter/robotic/scuttlebot))
```

Comment:
how is an item a mob

Reply:
Updated. Hopefully i stopped being dumb.